code/model/DMSDocument.php

```diff
@@ -914,6 +923,13 @@ public function getCMSFields()
         $fields->add(new TextField('Title', 'Title'));
         $fields->add(new TextareaField('Description', 'Description'));
 
+        $downloadBehaviorSource = array(
+            '' => 'Default ('.Config::inst()->get('DMSDocument', 'default_download_behaviour').')',
```

Comment:
As per my previous comment, the default behaviour needs to be saved as an explicit choice. You've just renamed "default" to "" which has the same problems.